app/controllers/works_controller.rb

```diff
@@ -87,7 +87,9 @@ def wait
   private
 
   def work_params
-    params.expect(work: WorkForm.user_editable_attributes + [WorkForm.nested_attributes])
+    # TODO: Extract nested form handling. See https://github.com/sul-dlss/hungry-hungry-hippo/issues/218
+    params.expect(work: WorkForm.user_editable_attributes + [WorkForm.nested_attributes] +
+    [{ publication_date_attributes: %i[year month day] }])
```

Comment:
Depending on your answer to my question above about what's currently broken w/ 1-to-1 nested attrs, I might want to have a go at making those changes before we move forward w/ this PR (*i.e.*, if the problem is small enough that I can hack it out in an hour or two). Reasonable?

Reply:
Once you get past the thick of the one-to-one work, I expect you won't need this anymore because `WorkForm.nested_attributes` will handle it for you.